tests/unit/test_simulated_stack.py

```diff
@@ -16,6 +16,9 @@
 import ga4gh.datamodel.sequenceAnnotations as features
 import ga4gh.frontend as frontend
 import ga4gh.protocol as protocol
+from ga4gh.allele_annotation_service_pb2 import \
```

Comment:
These two imports shouldn't be needed; just use ``protocol.SearchVariantAnnotationSetsResponse`` and ``protocol.GAException``